teal/block-parser.tl

```diff
@@ -511,9 +585,14 @@ parse_fns.fornum = function(state: ParseState, block: Block): Node
 end
 
 parse_fns.forin = function(state: ParseState, block: Block): Node
-   local node = new_node(state, block)
+   local node = new_node(state, block, "forin")
    node.vars = parse_variable_list(state, block[1])
    node.exps = parse_expression_list(state, block[2])
+   if #node.exps < 1 then
```

Comment:
What if `parse_expression_list` fails and returns `nil`?

Reply:
`parse_expression_list` should never return nil, but the `#exps == 0` can happen